test_postbodies/patch_merge.json

```
@@ -1,8 +1,9 @@
 {
   "mapsetlist": ["hpda_tiling_usermapset_tmp1", "hpda_tiling_usermapset_tmp2", "hpda_tiling_usermapset_tmp3"],
```

```suggestion
  "mapsetlist": ["hpda_tiling_usermapset_tmp1", "hpda_tiling_usermapset_tmp2", "hpda_tiling_usermapset_tmp3"],
```

Here, and elsewhere: I suggest to remove the "hpda_" prefixes which are unrelated to actinia_core.